data_provenance/README.md

```diff
@@ -15,6 +15,6 @@ The specific filter settings are here: https://github.com/Data-Provenance-Initia
 
 Here is the process to download the data, from inside the `data_provenance` dir:
 
-1. Run `python download.py --include include.csv`
+1. Run `python3 hf_downloader.py`
```

Comment:
I don't think we should specify python3 - we can assume that either users have python mapped to python3 or that they know they might need to do this (also to follow the convention with other readmes in this repo).

Reply:
Will do